backend/api/MQTT/MessageModels/IsarAvailability.cs

```diff
@@ -0,0 +1,49 @@
+using System.Text.Json.Serialization;
+using Api.Controllers.Models;
+using Api.Database.Models;
+
+namespace Api.Mqtt.MessageModels
+{
+#nullable disable
+    public class IsarAvailabilityMessage : MqttMessage
```

Comment:
This seem to have much more values than I would assosiate with an AvialabilityMessage. Should we limit this to a update on just the status? The other values seem more to be set on the initial connect

Reply:
The functionality is twofold in that it signals the availability of the ISAR instance but also enables Flotilla to discover new ISAR instances on the network. Thus all the information required to instantiate a robot object in the database must be available when the message is received. A suggestion from @Christdej (https://github.com/equinor/isar/pull/390) was to use an API call back to the ISAR instance to ask for more details instead. Currently undecided on which approach to go for.

Reply:
Ended up splitting in two topics with different event listeners for those two.